statsmodels/graphics/gofplots.py

```diff
@@ -178,20 +187,25 @@ def sorted_data(self):
         return sorted_data
 
     @cache_readonly
-    def sample_quantiles(self):
-        if self.fit and self.loc != 0 and self.scale != 1:
-            return (self.sorted_data-self.loc)/self.scale
+    def scaled_data(self):
+        if self.loc == 0 and self.scale == 1:
+            return (self.sorted_data-np.mean(self.data))/np.std(self.data)
```

Comment:
the log-normal example in the notebook will break here, if the user specifies loc=0 and scale=1 as the correct parameters.

If we want to have scaling of x-axis or data, then it should be an option. In terms of actual values used in the plot, this should be part of fitting or specifying parameters.